app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt

```diff
@@ -143,7 +147,8 @@ class AddSearchEngineFragment :
 
                 true
             }
-            else -> super.onOptionsItemSelected(item)
+            // other options are not handled by this menu provider
```

Comment:
I want to suggest dropping this comment. It should generally be understood by the `onMenuItemSelected` kDoc that a `false` returned value means the menu item option wasn't handled. I also don't want us to set a precedent where we have to add this comment for all the other `else -> false` in these `onMenuItemSelected`.

Reply:
Sure!

Reply:
Bumping this as I get back to reviewing and testing this